exporters/otlp/test/otlp_http_metric_exporter_test.cc

```diff
@@ -1,37 +1,52 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
+#include <gmock/gmock.h>
+#include <google/protobuf/message_lite.h>
```

Comment:
This header should be included between `protobuf_include_prefix.h` and `protobuf_include_suffix.h`

Reply:
Thanks,

moved `google/protobuf/message_lite.h` between `protobuf_include_prefix.h` and `protobuf_include_suffix.h`